Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:boron-alpine
```

Comment:
Why boron-alpine?

Reply:
Firstly. I added `boron`, but then changes suggested to use `boron-alpine`

Reply:
So, finally what should I add `boron` or `boron-alpine` ??

Reply:
I'm fine with boron-alpine, it finally boils down to docker optimization which we can look into it later. @bhaveshAn